applications/luci-app-filemanager/root/usr/share/rpcd/acl.d/luci-app-filemanager.json

```diff
@@ -0,0 +1,14 @@
+{
+	"luci-app-filemanager": {
+		"description": "Grant access to File Manager",
+		"write": {
```

Comment:
There are no permissions for ls if that's what you're going to use.

Reply:
(i.e. add some)

Reply:
As far as I understand what I need is 'exec' permission for "/*" With this it works and doesn't work without